packages/loopring_v3/contracts/lib/DelayedTransaction.sol

```diff
@@ -0,0 +1,321 @@
+/*
+
+  Copyright 2017 Loopring Project Ltd (Loopring Foundation).
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+pragma solidity ^0.5.11;
+
+import "../iface/IDelayedTransaction.sol";
+
+import "./AddressUtil.sol";
+import "./BytesUtil.sol";
+import "./MathUint.sol";
+
+
+/// @title DelayedOwner
+/// @author Brecht Devos - <brecht@loopring.org>
+/// @dev Base class for an Owner contract where certain functions have
+///      a mandatory delay for security purposes.
+contract DelayedTransaction is IDelayedTransaction
```

Comment:
How about we use the AddressSet's design to create a BytesSet, then each function will have its own bytes set for encoded DelayedFunction. I think this will simplify storage-related code in this contract.

Reply:
See #888